modules/gitops-release-notes-1-12-4.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-4_{context}"]
+= Release notes for {gitops-title} 1.12.4
+
+{gitops-title} 1.12.4 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.4_{context}"]
+== Errata updates
+
+[id="rhsa-2024:4163-gitops-1-12-4-security-update-advisory_{context}"]
+=== RHSA-2024:4163 - {gitops-title} 1.12.4 security update advisory
+
+Issued: 2024-06-27
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:4163[RHSA-2024:4163]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-12-4_{context}"]
+== Fixed issues
+
+* Before this update, the {gitops-title} Operator used an older version of the HA proxy router container image. This container image had multiple security vulnerabilities. With this update, the {gitops-title} Operator uses the latest version of the HA proxy container image, which includes a fix for the security vulnerabilities. link:https://issues.redhat.com/browse/GITOPS-4761[GITOPS-4761]
+
+* Before this update, the {gitops-title} Operator used an older version of Redis container image. This container image had multiple security vulnerabilities. With this update, the {gitops-title} Operator uses the correct version of the Redis container image, Redis 6, which includes a fix for the security vulnerabilities. link:https://issues.redhat.com/browse/GITOPS-3514[GITOPS-3514]
+
+* Before this update, the default TLS termination policy of the Argo CD server route changed from `passthrough` to `reencrypt`, leading to issues for users who previously configured the passthrough route with a custom TLS certificate. This update fixes the issue by reverting the default termination policy to `passthrough`. With this update, the Argo CD UI is accessible without any issues, even if the server route with a passthrough policy is configured with a custom TLS certificate. link:https://issues.redhat.com/browse/GITOPS-4758[GITOPS-4758]
```

Comment:
@Dhruv-Soni11 @varshab1210 Since we have reverted the policy to Passthrough, should we add a line to inform the users to manually delete the `argocd-server-tls` secret? This is already present in the [upstream docs](https://argocd-operator.readthedocs.io/en/latest/usage/routes/#tls-reencrypt-mode), but we should make it more explicit. I'll confirm this once.

Reply:
Nope. We've agreed not to include it since users are not impacted. Please feel free to mark this as resolved.